src/ProjectTemplates/BlazorTemplates.Tests/BlazorTemplates.Tests.csproj

```diff
@@ -25,6 +25,12 @@
     <TestDependsOnAspNetRuntime>true</TestDependsOnAspNetRuntime>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(UsingMicrosoftNETSdkWeb)' == 'true' OR '$(RazorSdkCurrentVersionProps)' != ''">
```

Comment:
This will always be false which is why it doesn't get imported from CSharp.Common.props. Let's remove this.

Reply:
Done.